src/pages/stablecoins.tsx

```diff
@@ -549,65 +319,85 @@ const StablecoinsPage = ({ data }: PageProps<Queries.StablecoinsPageQuery>) => {
         description={t("page-stablecoins-meta-description")}
       />
       <PageHero isReverse content={heroContent} />
-      {/* <HeroSectionContent>
-        <H2>
-          <Translation id="page-stablecoins-prices" />
-        </H2>
-        <p>
-          <Translation id="page-stablecoins-prices-definition" />{" "}
-          <Link to="#how">
-            <Translation id="page-stablecoins-prices-definition-how" />
-          </Link>
-        </p>
-        <H3>
-          <Translation id="page-stablecoins-coin-price-change" />
-        </H3>
-        [Insert graph showing price changes over 30days for stablecoins vs ETH
-        to demo comparative stability]
-      </HeroSectionContent> */}
       <Divider />
       <Content>
-        <TwoColumnContent>
-          <LeftColumn>
-            <H2>
+        <Flex
+          alignItems="flex-start"
+          width="100%"
+          mr="2rem"
+          mb="2rem"
+          flexDirection={{ base: "column", lg: "row" }}
+        >
+          <Box mr="2rem" w="100%" margin={{ base: "auto 0rem", lg: "" }}>
```

Comment:
Having `lg: ""` is weird. Have a look at the following approach.
```suggestion
          <Box w="100%" my="auto" mr={{ base: 0, lg: 8 }}>
```